scripts/qa_crowbarsetup.sh

```diff
@@ -3028,7 +3028,13 @@ function update_one_proposal()
     # hook for changing proposals:
     custom_configuration $proposal $proposaltypemapped
 
-    crowbar "$proposal" proposal commit $proposaltype
+
+    if iscloudver 6plus; then
+        safely crowbarctl proposal commit $proposal $proposaltype
+    else
+        crowbar "$proposal" proposal commit $proposaltype
```

Comment:
Now that we re-arranging code, should we get rid of the quotes around `$proposal` as you did in the previous line?